Src/AutoFakeItEasy.FakeItEasy2UnitTest/AutoFakeItEasy.FakeItEasy2UnitTest.csproj

```diff
@@ -8,15 +8,16 @@
     <AssemblyTitle>AutoFakeItEasy.FakeItEasy2.UnitTest</AssemblyTitle>
     <AssemblyName>AutoFixture.AutoFakeItEasy.FakeItEasy2UnitTest</AssemblyName>
     <RootNamespace>AutoFixture.AutoFakeItEasy.UnitTest</RootNamespace>
-    <DefineConstants>$(DefineConstants);CAN_FAKE_DELEGATES</DefineConstants>
+    <DefineConstants>$(DefineConstants);CAN_FAKE_DELEGATES;HAS_A_CALL_TO_SET</DefineConstants>
```

Comment:
I'd rename to `HAS_A_CALL_TO_SET_TESTER` (or whatever this kind of methods is called in your API), because first time I read it I was quite confused (probably, that's because I don't use FakeItEasy).